qcfractal/tests/test_storage.py

```diff
@@ -416,7 +416,7 @@ def test_empty_get(storage_results):
     assert 0 == len(storage_results.get_results(program='null')['data'])
 
 
-def test_results_query_total(storage_results):
+def test_results_get_total(storage_results):
 
     assert 6 == len(storage_results.get_results()["data"])
```

Comment:
These seem more like queries to me, I know that we are explicitly getting data, but its conditioned on broad terms, not specific fetches. No need to change them, but just a terminology confusion.

Reply:
They are, will look into storage rename after we switch out SQL.